src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml

```diff
@@ -28,12 +28,29 @@
         up static resources in the ${webappRoot}/resources directory -->
     <mvc:resources mapping="/resources/**" location="/resources/" />
 
+    <!--Add placeholder properties-->
+    <bean
```

Comment:
You should be able to configure this bean so it receives properties. Rather than duplicating the location.